.github/workflows/docs.yaml

```diff
@@ -29,7 +32,12 @@ jobs:
       - name: Fetch gh pages
         run: git fetch origin gh-pages --depth=1
       - name: Build docs
-        run: poetry run python tools/build_docs.py docs-build
+        run: |
```

Comment:
I'd make this into a separate step. Then it's easier to see from within the CI what's going on. So you'd have a `Build release docs` and a `Build dev docs` step